apps/studio/tests/setup/polyfills.ts

```diff
@@ -35,3 +35,5 @@ Object.defineProperties(globalThis, {
   ReadableStream: { value: ReadableStream },
   TransformStream: { value: TransformStream },
 })
+
+window.HTMLElement.prototype.hasPointerCapture = vi.fn()
```

Comment:
a click event on `SelectTrigger_shadcn_` calls this browser API, so we need to mock it